lib/reporters/teamcity_reporter.js

```diff
@@ -38,7 +38,9 @@ TeamcityReporter.prototype = {
     if (result.skipped) {
       this.out.write('##teamcity[testIgnored name=\'' + this._namify(prefix, result) + '\' message=\'pending\']\n');
     } else if (!result.passed) {
-      this.out.write('##teamcity[testFailed name=\'' + this._namify(prefix, result) + '\' message=\'' + escape(result.error.message) + '\' details=\'' + escape(result.error.stack) + '\']\n');
+      var message = (result.error && result.error.message) || '';
+      var stack = (result.error && result.error.stack) || '';
+      this.out.write('##teamcity[testFailed name=\'' + this._namify(prefix, result) + '\' message=\'' + escape(message) + '\' details=\'' + escape(stack) + '\']\n');
```

Comment:
It just renders empty message & details

```
##teamcity[testFailed name='PhantomJS 2.1 - Global error: ReferenceError: Can|'t find variable:' message='' details='']
``` 
https://github.com/testem/testem/pull/965/commits/2240b27090306e06b8b721f87c2d6879f86f80e3#diff-b0131a2a1e9b5b26022dfe678ebd3e6dR437